dataactvalidator/config/sqlrules/d43_detached_award_financial_assistance_2.sql

```diff
@@ -0,0 +1,37 @@
+--  If PrimaryPlaceOfPerformanceCode is not USA, Congressional District must be blank
+WITH detached_award_financial_assistance_d43_2_{0} AS
+    (SELECT submission_id,
+    	row_number,
+    	place_of_performance_zip4a,
+    	place_of_performance_congr
+    FROM detached_award_financial_assistance
+    WHERE submission_id = {0})
+SELECT
+    dafa.row_number,
+    dafa.place_of_performance_zip4a,
+    dafa.place_of_performance_congr
+FROM detached_award_financial_assistance_d43_2_{0} AS dafa
+WHERE 
+	CASE WHEN COALESCE(dafa.place_of_performance_zip4a, '') != ''
```

Comment:
This part doesn't need to be a CASE. We only need to check here that it's provided.